src/Arcus.Templates.WebApi/Properties/launchSettings.json

```diff
@@ -1,5 +1,16 @@
 {
   "profiles": {
+//#if (OpenApi)
+    "Arcus.Templates.WebApi": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "useSSL": false,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:4068/api/docs"
```

Comment:
Is 4068 the default arcus port ?

Reply:
Do we need applicationUrl or launchUrl ?

Reply:
Ah, was looking over that. Tested it with `applicationUrl`, but maybe `launchUrl` will work as well.

Reply:
Aha, was supposed to place a route in the `launchUrl` and a hostring URL in the `applicationUrl`.

Reply:
> Is 4068 the default arcus port ?

Yes, that's right. 👍

Reply:
Great!  I think we can close this.  Thanks for doing this so quickly